flexget/plugins/input/next_series_seasons.py

```diff
@@ -165,7 +165,8 @@ def on_search_complete(self, entry, task=None, identified_by=None, **kwargs):
                 else:
                     log.debug('%s %s was accepted, rerunning to look for next season.', entry['series_name'],
                               entry['series_id'])
-                    self.rerun_entries.append(self.search_entry(series, entry['series_season'] + 1, task))
+                    if not [entry for entry in self.rerun_entries if entry.get('series_season') == latest.season + 1]:
```

Comment:
I'd change this to use the `any` builtin instead of list comprehension since it's way more optimized for this type of check (will return `True` immediately when found and not iterate over the whole iterator)